docs/sources/enterprise/enterprise-configuration.md

```diff
@@ -304,3 +304,23 @@ Time until expiration for tokens which are renewed. Should have a value higher t
 ### lease_renewal_increment
 
 New duration for renewed tokens. Vault may be configured to ignore this value and impose a stricter limit.
+
+## [security.egress]
+
+> **Note:** Available in Grafana Enterprise v7.4+.
```

Comment:
Could you add a brief description of what these settings do? I'm guessing from the name something about outgoing security?